### PR TITLE
Implement the nightly functional test

### PR DESCRIPTION
**Pull request by tarekziade**
replaces https://github.com/elastic/connectors-python/pull/94

### CODE REVIEW

File: connectors/sources/tests/fixtures/mysql/Makefile [hunk 1]
```diff
@@ -1,31 +0,0 @@
-.PHONY: run-stack stop-stack load-data
```
Comment (artem-shelkovnikov) [inline connectors/sources/tests/fixtures/mysql/Makefile:1]: Why was this file removed? It's not needed any more?

Comment (tarekziade) [reply connectors/sources/tests/fixtures/mysql/Makefile:1]: it adds a lot of friction and requires to allow overwriting
- custom python
- custom venv

I am calling everything from `ftest` now and planning to do this for all fixtures.

so the "standard" would become a few python functions

Comment (artem-shelkovnikov) [reply connectors/sources/tests/fixtures/mysql/Makefile:1]: Cool, makes sense!

**Review by artem-shelkovnikov — APPROVED** [review]
